.compute

```diff
@@ -24,13 +24,14 @@ first=${nodes[0]}
 hostname=`hostname`
 # log timestamp prefix
 time_format='[%Y-%m-%d %H:%M:%.S]'
+threads=1
```

Comment:
This was a stupid name for distinguishing between node processes ("The script that runs on one allocated node") from the parallel processes it runs on one node (in case of `worker 0` and `ps`). 
Will rename it to sub-processes.